internal/services/relay/relay_namespace_resource.go

```diff
@@ -201,7 +201,7 @@ func resourceRelayNamespaceDelete(d *pluginsdk.ResourceData, meta interface{}) e
 	}
 
 	// we can't make use of the Future here due to a bug where 404 isn't tracked as Successful
-	log.Printf("[DEBUG] Waiting for Relay Namespace %q (Resource Group %q) to be deleted", id.Name, id.ResourceGroup)
+	log.Printf("[DEBUG] Waiting for Relay Namespace %q (Resource Group %q) to be deleted", id.NamespaceName, id.ResourceGroupName)
```

Comment:
fwiw this can become:

```suggestion
	log.Printf("[DEBUG] Waiting for %s to be deleted", *id)
```